src/ooxml/java/org/apache/poi/xwpf/usermodel/XWPFPictureData.java

```diff
@@ -146,7 +146,7 @@ public int getPictureType() {
         return 0;
     }
 
```

Comment:
with public APIs, we need to go through a deprecation cycle

Reply:
Is this a change that is desired to go through a deprecation cycle?